packages/@okta/vuepress-site/.vuepress/public/docs/api/postman/event-hooks.json

```diff
@@ -0,0 +1,235 @@
+          "raw": ""
+        },
+        "url": {
+          "raw": "{{url}}/api/v1/eventHooks/{{eventHookId}}/lifecycle/deactivate",
+          "host": [
+            "{{url}}"
+          ],
+          "path": [
+            "api",
+            "v1",
+            "eventHooks",
+            "{{eventHookId}}",
+            "lifecycle",
+            "deactivate"
+          ]
+        }
+      },
+      "response": []
+    },
+    {
+      "name": "Delete an event hook",
+      "request": {
+        "method": "DELETE",
+        "header": [
+          {
+            "key": "Authorization",
+            "value": "SSWS {{apikey}}"
+          },
+          {
+            "key": "Content-Type",
+            "value": "application/json"
+          }
+        ],
+        "body": {
+          "mode": "raw",
+          "raw": ""
+        },
+        "url": {
+          "raw": "{{url}}/api/v1/eventHooks/{{eventHookId}}",
+          "host": [
+            "{{url}}"
+          ],
+          "path": [
+            "api",
+            "v1",
+            "eventHooks",
+            "{{eventHookId}}"
+          ]
+        }
+      },
+      "response": []
+    },
+    {
+      "name": "Update an existing event hook",
+      "request": {
+        "method": "PUT",
+        "header": [
+          {
+            "key": "Authorization",
+            "value": "SSWS {{apikey}}"
+          },
+          {
+            "key": "Content-Type",
+            "value": "application/json"
+          }
+        ],
+        "body": {
+          "mode": "raw",
+          "raw": "{\n    \"status\": \"INACTIVE\",\n    \"verificationStatus\": \"VERIFIED\",\n    \"name\": \"Create, Activate and Login user event hook\",\n    \"events\": {\n        \"type\": \"EVENT_TYPE\",\n        \"items\": [\n            \"user.lifecycle.activate\",\n            \"user.lifecycle.create\",\n            \"user.session.start\"\n        ]\n    },\n    \"channel\": {\n        \"type\": \"HTTP\",\n        \"version\": \"1.0.0\",\n        \"config\": {\n            \"uri\": \"https://example.com\",\n            \"headers\": [\n                {\n                    \"key\": \"Authentication\",\n                    \"value\": \"some-api-key\"\n                },\n                {\n                    \"key\": \"X-Other-Header\",\n                    \"value\": \"some-other-value\"\n                }\n            ],\n            \"method\": \"POST\",\n            \"authScheme\": {\n                \"type\": \"HEADER\",\n                \"key\": \"Authorization\",\n                \"value\": \"Updated shared secret\"\n            }\n        }\n    }\n}"
+        },
+        "url": {
+          "raw": "{{url}}/api/v1/eventHooks/{{eventHookId}}",
+          "host": [
+            "{{url}}"
+          ],
+          "path": [
+            "api",
+            "v1",
+            "eventHooks",
+            "{{eventHookId}}"
+          ]
+        },
+        "description": "Update a full webhook (.e.g status or events)"
```

Comment:
The description should use `event hook`. Also, the recommendation is to use lifecycle methods for updating status, so we should remove `status` from the example here